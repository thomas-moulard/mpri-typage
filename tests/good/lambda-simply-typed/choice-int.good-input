val choice (f: (int -> int) -> (int -> int) -> int -> int) (x : int) =
  f (fun (x: int). x) (fun (x: int). x) x 
