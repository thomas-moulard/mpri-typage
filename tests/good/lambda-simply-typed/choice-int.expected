choice = fun (f : (int -> int) -> (int -> int) -> int -> int).fun (x : int).f (fun (x1 : int).x1) (fun (x1 : int).x1) x

choice : (((int -> int) -> ((int -> int) -> (int -> int))) -> (int -> int))
